### PR TITLE
(SIMP-6186) Address Freeradius Comments

### PR DESCRIPTION
**Pull request by jeannegreulich**
- Combined v3 configuration manifests under one directory, v3/conf.
- Updated LDAP erb so quotes were used consistently and fixed some
  errors in it.
- Created manifest for adding global listener.
- Added ability to remove local listener from ldap site so a global
  listener could be used.

SIMP-6186 #close

### CODE REVIEW

File: manifests/v3/listener.pp [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#
+# Create a global listener in the conf.d directory.
+#
+# @param order
+#  Indicates the order for this element amoung the concat fragments.
+#
+# The following parameters are all configuration parameters.
+# @see See /etc/raddb/radiusd.conf.sample for additional information.
```
Comment (lnemsick-simp) [inline manifests/v3/listener.pp]: Does this file exist?

File: spec/defines/v3/listener_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        let(:title) {'test1'}
+        let(:params) {{
+          :confdir     => '/etc/raddb',
+          :listen_type => 'auth',
+          :group       => 'radiusd'
+        }}
+
+        it { is_expected.to contain_concat('listener.test1') }
+        it { is_expected.to contain_freeradius__v3__listen('/etc/raddb/conf.d/listener.test1-fragment')
+#             .with({
```
Comment (lnemsick-simp) [inline spec/defines/v3/listener_spec.rb]: is there a reason this is commented out?

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,11 @@
+* Tue Mar 12 2019 Jeanne Greulich <jeanne.greulich@onyxpoint.com> - 8.0.0-0
```
Comment (lnemsick-simp) [inline CHANGELOG]: Please merge the three 8.0.0 changelog entries to have a cohesive 'this is what changed from 7.0.1'.  The little change I made doesn't warrant attribution.

File: manifests/v3/listener.pp (hunk 1 above)
Comment (jeannegreulich) [reply manifests/v3/listener.pp]: I fixed the @see examples and put a note in the README.

**Pull request merged**